### PR TITLE
[WIP] Re-design of the Downloads Page

### PR DESCRIPTION
**Pull request by logankilpatrick**
This will switch the overall page to be inline with the design we want across the website. 

See a preview at: https://julialang.netlify.app/downloads/

### CODE REVIEW

**Comment by ViralBShah** [comment]
Look at 1.5-beta. We need to incoproate MUSL binaries on Linux too.

I also think we should not list FreeBSD under Linux. Maybe just make it its own box.

**Comment by ViralBShah** [comment]
Will we then do this for all the 3 release types?

**Comment by logankilpatrick** [comment]
That’s the plan, just wanted initial feedback before I did it all.

**Comment by giordano** [comment]
A link to the build instructions for the "source code" link? https://github.com/JuliaLang/julia/tree/master/doc/build

**Comment by zdroid** [comment]
It looks like whitespace is missing between the sections, as everything is so tightly packed. You can add it with `.mb-{1,2,3,4,5}` classes from Bootstrap (`.mb-3` should be okay).

**Comment by giordano** [comment]
Does the Apple logo look a bit stretched only to me?

**Comment by zdroid** [comment]
@giordano Linux, FreeBSD and Apple logos on the page are not 1:1 originally, so when they get resized to 1:1, they look bad.

**Comment by ViralBShah** [comment]
Any thoughts on how to incorporate the LTS, stable, and pre-releases in this design? I don't think we want the OS logos to be replicated. So perhaps the logos stay on top, and then each OS gets three rows for each of the different releases. 

This would make it OS-centric rather than release-centric, which seems ok.

**Comment by logankilpatrick** [comment]
Yeah, I am looking at it right now with logos for each section and it's a bit too much.

**Comment by logankilpatrick** [comment]
Maybe we only need logos for the Stable release section as that is likely to be where most folks are going anyway.

**Comment by logankilpatrick** [comment]
Not sure we want it this bear bones but I actually like this design: 
Screen Shot 2020-06-17 at 7 12 34 PM

File: downloads/index.md [hunk 1]
```diff
@@ -161,56 +161,140 @@ Checksums for this release are available in both, [MD5](https://julialang-s3.jul
...
+  border: 2px solid #dddddd;
+  margin: 3px;
+  border-radius: 8px;
+}
+
+</style>
+
+<div class="grid-container">
+<!-- five columns -->
+	<div class="platform">
```
Comment (zdroid) [inline downloads/index.md:205]: Are tabs here intentional? Many HTML files in the website have this problem, of mixing tabs and spaces.

Comment (logankilpatrick) [reply downloads/index.md:205]: This code was graciously donated to me by @cormullion (not pointing fingers just highlighting that other people may use spaces while I use tabs most of the time) but we can clean all these up after the content and design are set.

Comment (ViralBShah) [reply downloads/index.md:205]: Yeah, we should indent it to be clean.

**Comment by logankilpatrick** [comment]
In the interest of not getting bogged down on this, I am going to move ahead with the logo download section for the stable release and then leave the other 2 the way they are now. I will also clean up the bottom section of the page to make some of those parts live in a subpage.

File: downloads/index.md (hunk 1 above)
Comment (cormullion) [reply downloads/index.md:205]: I plead guilty - my HTML formatting tool (BBEdit/TIdy) inserts tabs by default. 🤣

File: downloads/index.md [hunk 2]
```diff
@@ -138,7 +138,7 @@ Checksums for this release are available in both [MD5](https://julialang-s3.juli
   <!-- source -->
 
   <div class="downloads">
-  <p><a href="https://github.com/JuliaLang/julia/tree/v1.3.1">GitHub</a> </p>
+  <p><a href="https://github.com/JuliaLang/julia/tree/v1.4.2">GitHub</a> </p>
```
Comment (tlienart) [inline downloads/index.md:141]: should use `{{stable_release}}` for maintainability

**Comment by ViralBShah** [comment]
@logankilpatrick close or keep open?

**Pull request closed without merging**